ui/shared/tabs/failureSummary/SuggestionsListItem.jsx

```diff
@@ -110,7 +119,27 @@ export default class SuggestionsListItem extends React.Component {
           >
             <FontAwesomeIcon icon={faBug} title="File bug" />
           </Button>
-          <span>{suggestion.search}</span>
+          <span>{suggestion.search} </span>
+          <Clipboard
+            description=" text of error line"
+            text={suggestion.search}
+          />
+          <a
+            href={getLogViewerUrl(
+              selectedJob.id,
+              repoName,
+              suggestion.line_number + 1,
+            )}
+            target="_blank"
+            rel="noopener noreferrer"
+            title="Go to this line log viewer"
+          >
+            <img
+              alt="Logviewer"
+              src={logviewerIcon}
+              className="logviewer-icon pb-1 ml-1"
```

Comment:
We should remove the `pb-1` as it throws off the vertical alignment with the clipboard button.

Reply:
@camd, It is looking like this after removing the pb-1
There is a small white background around the alipboard icon due to whih it does not seem to be aligned so well
![Screenshot from 2020-06-29 21-23-56](https://user-images.githubusercontent.com/44131389/86027897-df03b900-ba4e-11ea-8232-33849b69465d.png)

Is it fine

Reply:
I think it looks fine.